### ISSUE TITLE
How to use  react-native-raw-bottom-sheet in typescript

### ISSUE THREAD

**Issue by kashif-sk** [nysamnang/react-native-raw-bottom-sheet#161]
I am trying to use the BottomSheet in my typescript project
 But getting the following error:
```
No overload matches this call.
  Overload 1 of 2, '(props: RBSheetProps | Readonly<RBSheetProps>): RBSheet', gave the following error.
    Type '{ children: Element; ref: RefObject<RBSheet>; closeOnDragDown: true; keyboardAvoidingViewEnabled: false; openDuration: number; animationType: "fade"; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<RBSheet> & Readonly<RBSheetProps>'.
      Property 'children' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<RBSheet> & Readonly<RBSheetProps>'.
  Overload 2 of 2, '(props: RBSheetProps, context: any): RBSheet', gave the following error.
    Type '{ children: Element; ref: RefObject<RBSheet>; closeOnDragDown: true; keyboardAvoidingViewEnabled: false; openDuration: number; animationType: "fade"; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<RBSheet> & Readonly<RBSheetProps>'.
      Property 'children' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<RBSheet> & Readonly<RBSheetProps>'.
```

Current Implementation is as follows:

```
const ref = useRef<RBSheet>(null);
  return (
    <RBSheet
      ref={ref}
      closeOnDragDown={true}
      height={300}
      keyboardAvoidingViewEnabled={false}
      openDuration={200}
      animationType={'fade'}>
      <View>
        <Text>Test</Text>
      </View>
    </RBSheet>
  );
```

**Comment by kashif-sk** [comment]
This PR fixes the issue [#146](https://github.com/nysamnang/react-native-raw-bottom-sheet/pull/146)

Using patch-package for now

**Comment by mashwishi** [comment]
still doesnt work.
